src/libs/merge/kdbmerge.c

```diff
@@ -0,0 +1,527 @@
+
+#include "kdbmerge.h"
+#include "kdb.h"
+#include "kdbassert.h"
+#include "kdblogger.h"
+#include "kdbprivate.h"
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+
+int nonOverlapOnlyBaseCounter = 0;  // Conflict where only base is different (it exists) and our and their are empty.
```

Comment:
global variables are absolute no-go in libraries as they would prohibit to use elektraMerge to be used concurrently.